library/bigip_selfip.py

```diff
@@ -526,11 +534,22 @@ def create(self):
             )
         else:
             vlan = "/%s/%s" % (partition, vlan)
-
         try:
-            ipin = "%s/%s" % (address, netmask)
-            ipnet = IPNetwork(ipin)
-            params['address'] = "%s/%s" % (ipnet.ip, ipnet.prefixlen)
+            # edited by manuadoor for rd feature
```

Comment:
these "edited by" comments are unnecessary since we know its part of your contribution.